dev/org.eclipse.codewind.ui/src/org/eclipse/codewind/ui/internal/actions/BindProjectAction.java

```diff
@@ -50,13 +50,25 @@
 
 	@Override
 	public void run(IAction action) {
+		CodewindConnection connection = null;
+
 		if (project == null) {
 			// Should not happen
 			Logger.logError("BindProjectAction ran but no project was selected"); //$NON-NLS-1$
 			return;
 		}
 		
-		CodewindConnection connection = setupConnection();
+		try {
+			if (CodewindInstall.isCodewindInstalled()) {
+				connection = setupConnection();
+			} else {
+				CodewindInstall.codewindInstallerDialog(project);
+				return;
+			}
+		} catch (InvocationTargetException e) {
+			Logger.logError(e);
```

Comment:
There should be some sort of message here explaining what it was trying to do when the exception happened such as:
Logger.logError("Error trying to set up Codewind to add existing project: " + project.getName(), e)